db.js

```diff
@@ -72,7 +72,7 @@ exports.init = function (sbot, config) {
   const state = {}
 
   sbot.close.hook(function (fn, args) {
-    close(() => {
+    close((err) => {
```

Comment:
This change does nothing right?

Reply:
I'm okay with leaving it in either case, just mostly trying to understand

Reply:
Yep, does nothing but I thought it was good to put there just to signal that we are ignoring the errors.

Reply:
Great, thanks